src/browser/ScriptManager.zig

```diff
@@ -136,14 +136,10 @@ pub fn addFromElement(self: *ScriptManager, element: *parser.Element) !void {
         if (std.ascii.eqlIgnoreCase(script_type, "module")) {
             break :blk .module;
         }
-        if (std.ascii.eqlIgnoreCase(script_type, "application/json")) {
-            return;
-        }
-        if (std.ascii.eqlIgnoreCase(script_type, "application/ld+json")) {
-            return;
-        }
```

Comment:
I'm not sure I understand why you remove these kind?
The won't create more log warning then?